package.json

```diff
@@ -12,11 +12,13 @@
     "url": "http://github.com/mozilla/browser.html/issues/"
   },
   "scripts": {
-    "server": "python -m SimpleHTTPServer 8080",
-    "start": "/Applications/B2G.app/Contents/MacOS/graphene --start-manifest=http://localhost:8080/manifest.webapp",
+    "start-server": "ecstatic . --port 6060",
```

Comment:
No npm creates `node_modules/.bin` folder and also includes it in path for `npm command` stuff so it's going to use the `ecstatic` that it installed and included in it's own path ;)